### DISCUSSION TITLE
Is it possible to exec npm script using ts-node?

### DISCUSSION THREAD

**Discussion by gerardolima** [TypeStrong/ts-node#1272] [Q&A] [answered]
I like `ts-node` and I'm trying to avoid using nodemon. Nodemon allows `nodemon ... --exec npm run test`. Is the same possible for `ts-node`

**Comment by cspotcode** [comment, marked as answer]
Typically this will be done by updating the npm scripts to use ts-node, rather than asking ts-node to invoke npm

Also, this might help:
https://github.com/TypeStrong/ts-node/discussions/1182

This env var can be used to install ts-node into any node process that receives it:
```
NODE_OPTIONS="-r ts-node/register"
```
